Content/StructureResolver.php

```diff
@@ -87,6 +94,18 @@ public function resolve(
             $data['view'][$property->getName()] = $contentView->getView();
         }
 
+        $titleProperty = $originalStructure->getProperty('title');
```

Comment:
think i would implement a private `getProperties` method with a `$requestedStructure` and `$targetStructure` argument. then we can loop over the properties of that method instead of `$structure->getProperties(true)` and the special logic for the `title` is handled at a single place